cristin-import/src/main/java/no/unit/nva/cristin/lambda/PublicationRepresentations.java

```diff
@@ -94,11 +97,24 @@ public boolean cristinObjectIsPartOfAnotherPublication() {
     }
 
     public boolean updateHasEffectiveChanges() {
-        var existingPublication = getExistingPublication().copy().withModifiedDate(null).withIdentifier(null).build();
-        var incomingPublication = getIncomingPublication().copy().withModifiedDate(null).withIdentifier(null).build();
+        var existingPublication = getExistingPublication().copy()
+                                      .withModifiedDate(null)
+                                      .withIdentifier(null)
+                                      .withImportDetails(getConcatenatedImportDetailsFromExistingAndIncomingPublication())
```

Comment:
Made life difficult for myself by not allowing removal of import details

Reply:
It seems like apache has a library for doing it, which is already in use in publication-api:

Try something like this:
`
!EqualsBuilder.reflectionEquals(existingPublication, incomingPublication, 
"modifiedDate", "identifier", "importDetails")
`

I think it is more elegant :)

Reply:
Couldn't get EqualsBuilder to actually ignore specified fields... I'll let this one be unless someone has another smoother ways to deal with this.

Reply:
Can ping me

Reply:
Conclusion: This method should be moved into the datamodel